test/Crowdfunding.js

```diff
@@ -232,13 +237,13 @@ describe("Crowdfunding", function () {
     
 
         it ("Should successfully contribute if everything is ok", async () => {
-
+          
 
             const tx = await crowdfunding.connect(address2).contribute(1, {value: etherToWei("1")});
             const result = await tx.wait();
             const event = result.logs[0].fragment.name;
 
-            expect(event).to.equal("ContributionCompleted");
+            expect(event).to.equal("ContributeCompleted");
```

Comment:
Let's keep `ContributionCompleted`, it should be a noun to be grammatically correct.

You can merge this tho, I have the change locally